src/components/Select/Select.tsx

```diff
@@ -311,6 +312,18 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
         return <EmptyOptions filter={filter} renderEmptyOptions={renderEmptyOptions} />;
     };
 
+    const _renderPopupFilter = () => {
```

Comment:
Select has `renderPopup` property, is it not enough?

Reply:
https://github.com/gravity-ui/uikit/blob/feat/select-zone/src/components/Sheet/SheetContent.tsx#L195

When using renderSelect, all content (both filters and the list) is placed inside .sheet-scroll-container.

Drawbacks:
- We have to override styles for deeply nested classes
- Logic for events like [onTouchStart](https://github.com/gravity-ui/uikit/blob/feat/select-zone/src/components/Sheet/SheetContent.tsx#L197) breaks

Reply:
Please show me what do you wanna do? I'll try to make it in sandbox

Reply:
@korvin89, filter and scrollable content 

1. https://preview.gravity-ui.com/uikit/?path=/docs/components-inputs-select--showcase&globals=platform:mobile
Story **With Custom Popup** + add height to .g-list

2. **With Virtualized List And Custom Popup**
https://preview.gravity-ui.com/uikit/2278/?path=/docs/components-inputs-select--showcase